configurations/kmip-ear.yaml

```diff
@@ -7,7 +7,7 @@ append_scylla_yaml: |
   system_info_encryption:
       enabled: True  # system_info_encryption
       key_provider: LocalFileSystemKeyProviderFactory  # system_info_encryption
-      secret_key_file: '/tmp/system_info_encryption_keyfile'
+      secret_key_file: '/etc/scylla/ssl_conf/system_info_encryption_keyfile'
```

Comment:
ssl_conf/ directory will only be created when server_encrypt is enabled.

In config_client_encrypt() we would remove the `/etc/scylla/ssl_conf/`. So it's dangerous the put the encryption_keyfile to it.

Let's recreate another directory and use it for ear key file. eg: `/etc/encrypt_conf/tmp/`

Reply:
probably the best will be `/etc/scylla/encrypt_conf`
why do we need "tmp" in that path?

Reply:
also, where is the directory originally created? (the current `/tmp...`)?

Reply:
i added a creation of that folder on `config_setup`, and now I didn't fail with the error on setup.

Reply:
@amoskong , my tests passed the whole setup, and started the c-s load.
i think it is ready to be merged then.

Reply:
@amoskong ?